src/regex.cr

```diff
@@ -557,6 +557,21 @@ class Regex
     match(str, pos, options: _options)
   end
 
+  # Matches a regular expression against `String` *str*. This starts at the character
```

Comment:
Minor detail: Mentioning the type is unnecessary here. That's expressed in the signature.
```suggestion
  # Matches a regular expression against *str*. This starts at the character
```